code/src/components/ThoughtList.js

```diff
@@ -0,0 +1,36 @@
+/* eslint-disable no-underscore-dangle */
+import React from 'react';
+
+const ThoughtList = ({ allThoughts, loading, onThoughtChange }) => {
+  if (loading) {
+    return (
+      <h2>Loading..</h2>
+    )
+  }
+
+  return (
+    <section className="thought-list">
+      {allThoughts.map((thought) => {
+        return (
+          <div key={thought._id} className="thought-message">
+            <h3>{thought.message} </h3>
+            <div className="like-container">
+              <button
+                type="button"
+                className="btn-like"
+                onClick={() => onThoughtChange(thought._id)}
+                style={{
+                  background: thought.hearts >= 1 ? '#ffadad' : '#f2f2f2'
+                }}>
+                ❤️
+              </button>
+              <p className="likes"> {thought.hearts} </p>
+            </div>
+          </div>
+        )
```

Comment:
if you want to add the time of when they have posted the thought to the API you can add following : 
**<p className="date">{formatDistance(new Date(thought.createdAt), Date.now(), { addSuffix: true })} </p>**

You need to add the formatDistance to formate the date/time from the API so it will be shown as 15 minutes ago. 

You also need to **install date-fns** in your terminal ( make sure you are in the code folder) and then in the top of this component import :
**import { formatDistance } from 'date-fns'**